stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -964,6 +965,82 @@ Return the updated `y`.
 """
 sbmv!
 
+### spmv!, (SP) symmetric packed matrix-vector operation defined as y := alpha*A*x + beta*y.
+for (fname, elty) in ((:dspmv_, :Float64),
+                      (:sspmv_, :Float32))
+    @eval begin
+        function spmv!(uplo::AbstractChar,
```

Comment:
Could you add here the original BLAS signature here for completeness. You have it also in `hpmv!`.